src/index.ts

```diff
@@ -1,5 +1,99 @@
-const mapData = (data: any): any => {
-  return data
-}
+import {
+  Data,
+  SingleData,
+  PropsData,
+  PropsSingleData,
+  Gender,
+  Size,
+  Direction,
+} from "./types";
 
-export default mapData;
\ No newline at end of file
+let email: string | null;
+let phone: number | null;
+
+const isEmail = (email: string): boolean => email.includes("@");
+
+const convertGender = (gender: string): Gender => {
+  if (gender === "M") {
+    return "Male";
+  } else {
+    return "Female";
+  }
+};
+
+const handleComponentSize = (component: HTMLElement): string => {
+  const sizes: Size[] = ["small", "medium", "large"];
+  const [currentSize] = sizes.filter((size) => {
+    if (component.classList.contains(size)) return size;
+  });
+  return currentSize;
+};
+
+const onButtonClick = (direction: Direction, component: HTMLElement): void => {
+  const currentSize: string = handleComponentSize(component);
+
+  if (direction === "grow") {
+    switch (currentSize) {
```

Comment:
brakuje opcji `default`. Ona co prawda nic nie powinna zrobić więc nie jest to błąd, ale dobrą praktyką jest zawsze dorzucenie dyrektywy `default`